packages/lib/test/index.test.ts

```diff
@@ -1,494 +1,532 @@
-import chai, { expect } from 'chai'
+import * as chai from "chai";
```

Comment:
In this file it seems that a semicolon in inserted after every line and single quotes are replaced by double quotes. We might have to run prettier again to remove semicolons and go back to single quotes